backend/services/implementations/__tests__/classService.test.ts

```diff
@@ -134,6 +135,18 @@ describe("mongo classService", (): void => {
     );
   });
 
+  it("getClassesByTeacherId for valid testSessionId", async () => {
+    const savedClasses = [await ClassModel.create(testClassWithTestSessions)];
+    const res = await classService.getClassesByTeacherId(mockTeacher.id);
+    assertArrayResponseMatchesExpected(savedClasses, res);
```

Comment:
```suggestion
    const savedClass = await ClassModel.create(testClassWithTestSessions);
    const res = await classService.getClassesByTeacherId(savedClass.teacher);
    assertArrayResponseMatchesExpected([savedClass], res);
```
that looked weird to me for some reason lol SORRY. also just changed mockTeacher.id -> savedClass.teacher for clarity

Reply:
ah that looks better chief